### ISSUE TITLE
GitHub integration

### ISSUE THREAD

**Issue by greenbagels** [uncled1023/Combot#2]
Perhaps you could add in a github repo monitoring feature or a github searching feature?

**Comment by uncled1023** [comment]
The search is simple to do.  For the monitoring, do you mean having the bot subscribe to events of a repo and displaying them in the channel?

**Comment by greenbagels** [comment]
Yeah, so it can post any issues or PRs and merge events as they occur.

**Comment by uncled1023** [comment]
Hm, ok.  I will have to think of a way so that someone can't subscribe to a repo and have it spam notifications into a channel.

**Comment by uncled1023** [comment]
I have added github user and repo searching in commit https://github.com/uncled1023/Combot/commit/ef8b14906dbe1a4651d67065d3fc38144af98fa4.
Will look into hooks.
